docker-runner.mjs

```diff
@@ -0,0 +1,31 @@
+import {spawn} from 'child_process';
+
+let grist;
+
+function startGrist(newEnv) {
+  const env = newEnv ? {...process.env, ...newEnv} : process.env;
+
+  // H/T https://stackoverflow.com/a/36995148/11352427
+  grist = spawn('./sandbox/run.sh', {
```

Comment:
Does this get killed correctly when parent is killed? I'd think so, but just want to double check :slightly_smiling_face:

Reply:
Yep! It works! That's because the `run.sh` process `exec`s into node, so this `spawn` effectively ends up directly controlling a node process.